src/mdx-pages/guides/options.mdx

```diff
@@ -615,6 +615,20 @@ Override the mute key definition. If this is set, the function receives the `key
 
 Override the play/pause key definition. If this is set, the function receives the `keydown` event; if the function returns `true`, then the play/pause toggle action is performed.
 
+### `useSVGIcons`
+
+> Type: `boolean`
+
+_Since 8.4.0_
```

Comment:
This is assuming this gets released in the next version, but will be changed if that is not the case.